src/main/java/AutomationTrialQA2_multilogin/OpenBrowser.java

```diff
@@ -0,0 +1,313 @@
+package AutomationTrialQA2_multilogin;
+
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.concurrent.TimeUnit;
+
+import org.junit.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import AutomationTrialQA2_copy.BaseBrowser;
+
+
+
+public class OpenBrowser extends BaseBrowser {
+	
+	
+	public static void main(String[] args, String Username, String Password, String clearurl) throws InterruptedException {
+		OpenBrowser OB=new OpenBrowser();
+		
+		
+		OB.startBrowser1();
+		//OB.clearData(clearurl);
+		OB.hitCBTURL();
+		OB.login1(Username,Password);
+		OB.gothroughInstructions();
+		OB.logout();
+		
+	}
+	public void startBrowser1() throws InterruptedException {
+		
+		startBrowser();
+	}
+	public void clearData(String clearurl) throws InterruptedException{
+		driver.manage().window().maximize();
+		//driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
+		//driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);  
+		//driver.manage().timeouts().getImplicitWaitTimeout();
+		
+		//wait = new WebDriverWait(driver, 15);
+		
+		
+		driver.get("https://qa.edulabcbt.com/oneup/auth/clearExamData?email="+clearurl);
+		//driver.get(clearurl);
+		Thread.sleep(1000);
```

Comment:
Why you have use thread.sleep?

Reply:
because there clearurl is different from our application it clears the exam which is completed so we required static time for that so thread.sleep(1000) use to execute that url .

Reply:
ok